invenio/modules/deposit/testsuite/test_deposit_models.py

```diff
@@ -57,9 +63,13 @@ def test_create(self):
         d = Deposition.create(user)
         assert d.type == self.DefaultType
         assert Deposition.get(d.id).type == self.DefaultType
-        d = Deposition.create(user, type=self.AnotherType)
-        assert d.type == self.AnotherType
-        assert Deposition.get(d.id).type == self.AnotherType
+        d2 = Deposition.create(user, type=self.AnotherType)
+        assert d2.type == self.AnotherType
+        assert Deposition.get(d2.id).type == self.AnotherType
+
+        # remove the records
+        Deposition.delete(d)
```

Comment:
Similar except that above tests Deposit/Workflow integration, and in the example above you only have one BibWorkflowObject, whereas you can normally have many.

It's from experience that this is a soft spot between the two modules.